cegs_portal/search/templates/search/v1/dna_feature.html

```diff
@@ -607,7 +608,7 @@ <h1>
             {% for feature in features %}
             {% if feature.closest_features.exists %}
                     {% for close_feature in feature.closest_features.all %}
-                    <tr>
+                    <tr data-href="{% url 'search:dna_features' 'accession' close_feature.accession_id %}">
```

Comment:
does this need the javascript to set the click handler?

Reply:
Yes